src/runtime_src/core/pcie/emulation/cpu_em/generic_pcie_hal2/shim.cxx

```diff
@@ -556,6 +559,14 @@ namespace xclcpuemhal2 {
 
     bool isVersal = false;
     std::string binaryDirectory("");
+    
+    //Check if device_process.log already exists. Remove if exists.
+    std::string extIoTxtFile = deviceDirectory + "/../../../device_process.log";
+    FILE* fp = fopen(extIoTxtFile.c_str(), "rb");
+    if (fp != NULL)
```

Comment:
NULL is not C++, use nullptr, but moot once you use C++.

Reply:
This is resolved by using std::filesystem::exists and remove.